### ISSUE TITLE
learn how to export the Active Ingredient facet to its own Table

### ISSUE THREAD

**Issue by mroswell** [mroswell/list-N#14]
probably doing this manually. But nice to know.

**Comment by mroswell** [comment]
Maybe related to: https://simonwillison.net/2017/Nov/25/new-in-datasette/ 
Even so, I'm not sure the -c option would handle the array.

**Comment by mroswell** [comment]
10 minutes into: https://www.youtube.com/watch?v=7kDFBnXaw-c
sqlite-utils extract trees.db trees qLegalStatus

**Comment by mroswell** [comment]
In theory, could use this, (YouTube above, or documentation here: https://sqlite-utils.datasette.io/en/latest/cli.html#extracting-columns-into-a-separate-table
but given the arrayed version of my data, just doesn't seem useful. I ended up just incorporating the Safer and Toxic list into the python code. I generated the initial lists of toxic vs. safer by first creating a CSV, and processing as follows:

```
with open('./data/ActiveIngredientsSafeOrNot.csv','rb') as csvfile:
    reader = csv.reader(csvfile, delimiter=',', quotechar='|')
    for row in reader:
        if row[1] == 'Y':
            print row[0]
            safer_disinfectants.append(row[0])
print(safer_disinfectants)
```
(same for toxic_disinfectants)

Then incorporated those lists in transform.py
